lib/install.js

```diff
@@ -56,11 +56,13 @@ function place_binary(uri, targetDir, opts, callback) {
                     process.env.npm_config_proxy;
   let agent;
   if (proxyUrl) {
-    const ProxyAgent = require('https-proxy-agent');
-    agent = new ProxyAgent(proxyUrl);
+    // TODO: replace with undici's ProxyAgent
```

Comment:
I have some minor concerns about using a different proxy agent that I will bring up in https://github.com/mapbox/node-pre-gyp/pull/743.

The gist is that `npm` and `node-gyp` use the same http stack by design so that historically things like env vars are picked up the exact same. These concerns might be outdated and I know at least this specific one about env vars is.

My only feedback in this PR is to remove these comments and we can address all of it in the other PR.

Reply:
updated. thanks!